project.clj

```diff
@@ -24,32 +24,32 @@
                  [com.cemerick/url "0.1.1" :exclusions [com.cemerick/clojurescript.test]]
                  [org.cyverse/dire "0.5.6"]
                  [me.raynes/fs "1.4.6"]
-                 [org.apache.tika/tika-core "2.9.2" :exclusions [org.slf4j/slf4j-api]]
+                 [org.apache.tika/tika-core "3.2.2" :exclusions [org.slf4j/slf4j-api]]
```

Comment:
Do we know what updates with this major version update? Seems sort of aside the jargon update, but we also don't use that much of Tika so it might be fine. If we update it here we should in heuristomancer/info-typer/bulk-typer too though, but those probably use jargon anyway so they're probably in the list.

Reply:
There are some breaking changes between the two versions, but I didn't see anything that obviously affects us. You can see the list of changes here: https://dist.apache.org/repos/dist/release/tika/3.2.2/CHANGES-3.2.2.txt.

Updating heuristomancer, info-typer, and bulk-typer is a good idea as well. Info-typer and bulk-typer were on the list already, heuristomancer wasn't.

Reply:
Sounds good. Looking at that list, the change from returning `application/javascript` to returning `text/javascript` was the only thing that stuck out to me. However, it looks like Sonora just ignores the first part of the mimetype when matching, so since that's unchanged it shouldn't affect the behavior of the text editor/viewer/etc if I haven't missed something.